app/src/main/java/kr/co/softcampus/sopt_assignment1/HomeActivity.kt

```diff
@@ -0,0 +1,32 @@
+package kr.co.softcampus.sopt_assignment1
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import kr.co.softcampus.sopt_assignment1.databinding.ActivityHomeBinding
+
+class HomeActivity : AppCompatActivity() {
+
+    private lateinit var binding: ActivityHomeBinding
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        binding.git.setOnClickListener {
+            val url = "https://github.com/ChoiYuLim"
+            val webPage: Uri = Uri.parse(url)
```

Comment:
코틀린에서는 타입추론 기능을 지원하기 때문에 그냥
```suggestion
            val webPage = Uri.parse(url)
```
이렇게만 작성해도 상관없습니다~!!

Reply:
아! 이게 코틀린의 큰 장점이군요~~